priv/repo/migrations/00000000000001_create_user.exs

```diff
@@ -9,7 +9,6 @@ defmodule Org.Repo.Migrations.CreateUser do
       add :login, :string
       add :company, :string
       add :location, :string
-      add :created_at, :string
```

Comment:
Actually, this is here because it keeps a record of when the github user was created. I figured it would be useful in case we ever want to detect if someone just created an account 10 minutes earlier to get access.